tests/unit/testplan/testing/multitest/test_result.py

```diff
@@ -404,6 +404,37 @@ def test_flattened_comparison_result(self, dict_ns):
             and tea_1[4][0] == "REGEX"
         )
 
+    def test_exclude_key_on_nested_object(self, dict_ns):
```

Comment:
how about to add tests that are having actual differences and so ignored failed comparisons.

Reply:
yeah, I extended this testcase, adding nested objects which has sub keys that have different values.